File: src/main/java/org/saigon/striker/config/WebConfig.java
package org.saigon.striker.config;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.context.annotation.Import;
import org.springframework.web.reactive.config.WebFluxConfigurer;

@Configuration
@Import(SecurityConfig.class)
public class WebConfig implements WebFluxConfigurer {

    @Bean
    public DefaultErrorResponse defaultErrorResponse() {
        return new DefaultErrorResponse();
    }
}

File: src/main/kotlin/org/saigon/striker/config/WebConfig.kt
package org.saigon.striker.config

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.context.annotation.Import
import org.springframework.web.reactive.config.WebFluxConfigurer

@Configuration
@Import(SecurityConfig::class)
class WebConfig : WebFluxConfigurer {

    @Bean
    fun defaultErrorResponse(): DefaultErrorResponse = DefaultErrorResponse()
}
